lib/cosmos/utilities/sleeper.rb

```diff
@@ -0,0 +1,37 @@
+# encoding: ascii-8bit
+
+# Copyright 2014 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+module Cosmos
+
+  class Sleeper
```

Comment:
I'm going to need some documentation in here. What does this class do and why is it everywhere now?